go-client/TestSetupAbstraction_test.go

```diff
@@ -162,7 +162,7 @@ func (suite *TestSetupAbstractionSuite) TestDeleteFile() {
 
 func (suite *TestSetupAbstractionSuite) TestDeleteNonExistingFileSucceeds() {
```

Comment:
Shouldn't the method then be renamed to something like:

```suggestion
func (suite *TestSetupAbstractionSuite) TestDeleteNonExistingFileReturnsError() {
```